### DISCUSSION TITLE
빵빠레 울리고 따로 뭐를 눌러야하나요?

### DISCUSSION THREAD

**Discussion by yeyetim** [changdoc/naver-vaccine-macro#99] [Q&A]
빵빠레가 울리면 빠르게 예약신청을 눌러야하나요? 아니면 이것도 자동인가요? 

먼저 이런 좋은 프로그램을 개발해주셔서 깊은 감사드립니다

**Comment by koolingo** [comment]
빵빠레 울리고 자동으로 예약까지 진행 합니다.
실패시에는  잔여백신 없음으로 나옵니다.
그러면 다시 병원 지정한후에 예약 시작 하셔야 합니다.
